dev-manuals/index.rst

```diff
@@ -84,3 +84,10 @@ Developer Manuals
    :caption: Miscellaneous
 
    ./etc/**
+
+.. toctree::
```

Comment:
Please move this up.
Apart from `etc`, I'd like to keep the alphabetic sorting.